### ISSUE TITLE
[Audio Player]: QA: Before playing the audio, one unwanted extra tab focus is observed at the end of the player

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9288]
### Description

Page Element: Before playing the sound, at the end of the player

Issue Description:
one unwanted extra tab focus is observed at the end of the player. Focus is actually going to the slider button which is disabled.

Expected:Unwanted tab stop should be removed.

User Impact:
When standard methods of keyboard navigation is not supported, this impact the user ability to interact with the page controls. This impacts Keyboard only users/Screen reader users

### Component(s) impacted

Audio Player

### Browser

Firefox

### Carbon for IBM.com version

v1.37.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/react-experimental/?path=/story/components-audio-player--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
